conftest.py

```diff
@@ -62,6 +63,27 @@ def professional(profile3):
                                        description='I')
 
 
+@pytest.fixture
+def typeOfJob(professional):
+    return TypeOfJob(professional_id=professional, typeOfJob_name="Gel nail polish", price=90)
+
+
+@pytest.fixture
+def appointment(typeOfJob, professional, client):
+    return Appointment(professional_id=professional, client_id=client, typeOfJob_id=typeOfJob,
+                       start_appointment=datetime(2023, 4, 17, 12, 0, 0),
```

Comment:
Try to use relative times instead, like (now - 5 days). Decreases maintenance in tests' dates

Reply:
Consider the same approach in other places as well...

Reply:
What is the problem with the way I did?
I can't understand the advantage of the way you suggested.
I need specific hours for the tests

Reply:
Since it will not give you warning on pytest that the dates has passed already
So whenever you use time, use the now-Xdays to avid the errors with out dependency of hardcoded time

Reply:
Done